proxy/start_container_interceptor.go

```diff
@@ -11,7 +11,24 @@ import (
 
 type startContainerInterceptor struct{ proxy *Proxy }
 
+func callWeaveAndLog(container *docker.Container, description string, args ...string) error {
+	if _, stderr, err := callWeave(args...); err != nil {
+		return fmt.Errorf("Failed: %s container %s: %s", description, container.ID, string(stderr))
+	} else if len(stderr) > 0 {
+		Log.Warningf("%s container %s: %s", description, container.ID, string(stderr))
+	}
+	return nil
+}
+
```

Comment:
Those log messages look garbled to me. Won't we end up with "Failed: Attaching to weave network container 12345: someError", and "Failed: Notifying weave of container restart container 12345: someError"?